src/DiagramGenerator/Image/Image.php

```diff
@@ -18,13 +19,17 @@ class Image
     /** @var BaseImage */
     protected $image;
 
-    /** @var Storage */
+    /** @var StorageInterface */
     protected $storage;
 
     /** @var Config */
     protected $config;
 
-    public function __construct(Storage $storage, Config $config)
+    /**
+     * @param StorageInterface $storage
+     * @param Config $config
+     */
+    public function __construct($storage, Config $config)
```

Comment:
Any reason type hint is removed from storage?
```suggestion
    public function __construct(StorageInterface $storage, Config $config)
```